### PR TITLE
Add Hozo, Hozo 3Dprinter

### PR DESCRIPTION
**Pull request by jameschen1605**
Add Hozo, Hozo 3Dprinter from HardPrint

### CODE REVIEW

**Comment by jameschen1605** [comment]
Add definition, extruder and hotbed model.

File: resources/definitions/HardPrint_Hozo.def.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
```
Comment (Ghostkeeper) [inline resources/definitions/HardPrint_Hozo.def.json:1]: Could you change the file name, and IDs referring to it, to lowercase like the rest of the printer definitions?

File: resources/definitions/HardPrint_Hozo.def.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    "version": 2,
+    "name": "HardPrint Hozo, Hozo",
```
Comment (Ghostkeeper) [inline resources/definitions/HardPrint_Hozo.def.json:3]: I was unable to find any information about this printer. Is it being sold anywhere? Could you provide a link to a sales page?

File: resources/definitions/HardPrint_Hozo_iX.def.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
+{
```
Comment (Ghostkeeper) [inline resources/definitions/HardPrint_Hozo_iX.def.json:1]: Could you change the file name, and any IDs referring to it, to be lowercase, like the rest of the printer definitions?

**Review by Ghostkeeper — CHANGES_REQUESTED** [review]
Thanks for the profile! I have a few things I'd like to get cleared up before we can add it though:

File: resources/definitions/HardPrint_Hozo.def.json (hunk 2 above)
Comment (jameschen1605) [reply resources/definitions/HardPrint_Hozo.def.json:3]: Hello, we are Hardprint Co, an entrepreneurial team from Taiwan, currently stationed in Taiwan’s largest Maker Space M.zone.
Hozo, Hozo and Hozo iX are our DIY educational 3D printers, which are expected to be promoted and sold in February. Below is a link to our promotional video.

https://fb.watch/anBOWpgUwC/

Thank you very much

File: resources/definitions/HardPrint_Hozo.def.json (hunk 1 above)
Comment (jameschen1605) [reply resources/definitions/HardPrint_Hozo.def.json]: ok, I fixed it.

**Comment by Ghostkeeper** [comment]
We can add them in February then. We don't typically publish profiles for printers that are still in development.

**Comment by jameschen1605** [comment]
Thank you very much for your support and help.

**Comment by jameschen1605** [comment]
Sorry, I don't know what went wrong.

File: resources/extruders/hardprint_hozo_extruder_0.def.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
+{
+    "version": 2,
+    "name": "Extruder 1",
+    "inherits": "fdmextruder",
+    "metadata": {
+        "machine": "HardPrint_Hozo",
```
Comment (Ghostkeeper) [inline resources/extruders/hardprint_hozo_extruder_0.def.json:6]:
```suggestion
        "machine": "hardprint_hozo",
```

File: resources/extruders/hardprint_hozo_ix_extruder_0.def.json [hunk 5]
```diff
@@ -0,0 +1,15 @@
+{
+    "version": 2,
+    "name": "Extruder 1",
+    "inherits": "fdmextruder",
+    "metadata": {
+        "machine": "HardPrint_Hozo_iX",
```
Comment (Ghostkeeper) [inline resources/extruders/hardprint_hozo_ix_extruder_0.def.json:6]:
```suggestion
        "machine": "hardprint_hozo_ix",
```

**Review by Ghostkeeper — CHANGES_REQUESTED** [review]
The failing tests are about IDs that aren't updated when you changed the file names. These refer to each other:

**Comment by Ghostkeeper** [comment]
The currently failing tests are from our new build environment and the Python 3.10 merge, not the fault of this PR.

**Review by Ghostkeeper — APPROVED** [review]
Looks good now. I'll give it a quick test.

**Pull request merged**